castle/cms/_scripts/create-site.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import json
+import os
+
+import transaction
+from AccessControl.SecurityManagement import newSecurityManager
+from castle.cms.cron.utils import setup_site
+
+
+try:
+    from castle.cms.browser.site.addsite import AddCastleSite
+except ImportError:
+    # required! This could be getting called from upgrade
+    # integration tests
+    from castle.cms.browser.addsite import AddCastleSite
+
+
+app = app  # noqa
```

Comment:
?

Reply:
app is magic global variable inserted into script. This is to trick linter to not complain about future refs to global in code.